examples/stm32/WolfSSL/.project

```diff
@@ -320,5 +320,10 @@
 			<type>1</type>
 			<locationURI>PARENT-3-PROJECT_LOC/src/import/tls/wolfssl/ctaocrypt/src/wolfcrypt_last.c</locationURI>
 		</link>
+		<link>
+			<name>wolfmath.c</name>
+			<type>1</type>
+			<locationURI>PARENT-3-PROJECT_LOC/src/import/tls/wolfssl/wolfcrypt/src/wolfmath.c</locationURI>
```

Comment:
not sure.  Again, don't know if I'm fighting broken project or new wolfssl version here, but it failed to compile without this.